dumper/app/src/main/resources/oracle-stats/awr/hist-cmd-types-awr.sql

```diff
@@ -0,0 +1,46 @@
+-- Copyright 2022-2024 Google LLC
+-- Copyright 2013-2021 CompilerWorks
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+SELECT
```

Comment:
Can you compare this file to the existing statspack `hist-cmd-types`? I think there are some differences in column naming and we should keep it consistent.

Reply:
The other hist-cmd-types is older, from before there was any feedback from Assessment. It contains some inaccurate names and redundant data, so I'd like to adjust the other file and keep the names in here.